src/PDE/Integrate/Volume.C

```diff
@@ -321,13 +321,12 @@ tk::volIntP2( ncomp_t system,
     // Gaussian quadrature
     for (std::size_t igp=0; igp<NG; ++igp)
     {
-      // Continue to calculate the derivatives of the basis functions dB/dx for DG(P2)
+      // Calculate derivatives of the basis functions dB/dx for DG(P2)
       auto db5dxi1 = 12.0 * coordgp[0][igp] + 6.0 * coordgp[1][igp]
                    +  6.0 * coordgp[2][igp] - 6.0;
       auto db5dxi2 =  6.0 * coordgp[0][igp] + 2.0 * coordgp[1][igp]
                    +  2.0 * coordgp[2][igp] - 2.0;
-      auto db5dxi3 =  6.0 * coordgp[0][igp] + 2.0 * coordgp[1][igp]
-                   +  2.0 * coordgp[2][igp] - 2.0;
+      auto db5dxi3 = db5dxi2;   // = 6.0 * xi1 + 2.0 * xi2 + 2.0 * xi3 - 2.0
```

Comment:
I have such mixed feelings about this, do we think it's the right way to handle this?

On one side, explicitly donating the third term is equal to the second, and still referencing the third term later seems sensible...

On the other, a line that just is `terse_variable_a = terse_variable_b` (with some actual very terse variable) looks like a problem waiting to happen